bin/tart2ms

```diff
@@ -48,12 +48,19 @@ if __name__ == "__main__":
                             instantaneous-zenith    : Default options. Each snapshot is a separate field
                             rephase-obs-midpoint    : --rephase obs-midpoint --single-field
                             no-rephase-obs-midpoint : --single-field   
-                            rephase-SCP             : --rephase SCP --single-field
-                            rephase-NCP             : --rephase NCP --single-field
+                            <another known position>: --rephase <known position> --single-field
+    
+    Other known positions are read from named_phasings and from catalogs e.g. 3C sources
+    They may also be special bodies like Sun or Moon.
     '''
+    catalog_positions = catalog_factory.from_3CRR(fluxlim15=0.0)
+    named_positions = util.read_known_phasings()
+    specpos = list(map(lambda s: s.name, catalog_positions)) + \
+              list(map(lambda s: s['name'], named_positions))
     parser.add_argument('--rephase', required=False, default=None,
-                        choices=['obs-midpoint', 'SCP', 'NCP'],
-                        help="Rephase all visibilities to a new phase center")
+                        help=f"Rephase all visibilities to a new phase center. 'obs-midpoint' rephases to the zenith at the observation "
+                             f"midpoint for the provided databases. Otherwise another known position can be given or a twelve digit J2000 epoch coordinate "
+                             f"(e.g. J193900-632400). Currently recognized special positions are: {','.join(specpos)}")
```

Comment:
I'm going to insist if we use J or B multidigit formats to stick to the standard way of defining them which is non-truncated. The regex therefore is strictly J|B[0-9]{6}+|-[0-9]{6}. We can add GAL<float>+|-<float> later

Reply:
The standard way of using them in publications is the truncated way. In fact the MeerKAT gaincal list uses the truncated name as the primary key! 

The truncated name is far more useful than  named phasing (which we do) because these are the standard names, and  they provide co-ordinates with sufficient accuracy for TART use. For example J1939-6342 is the widely used name.

J193925-634246 returns ZERO google scholar hits. So it has never been used in a recent publication.
J1939-6342 returns HEAPS of them (https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=J1939-6342&btnG=)

Why force people to go to a catalogue to find a source location (or even worse, add  spurious zeros !!! as you've done in the example :).  We should not be in the business of including catalogues in our source code, and this eliminates it almost entirely (except for non-fixed objects)

Reply:
I can possibly allow the 8 digit convention ie. hhmm{ss}?+|-ddmm{ss}. 

The catalogs with J names are there only for source model prediction and to augment he sky with positions of bright things and their fluxes so we can scale the flux accordingly. If the user makes long integration maps. We should be able to resolve names such as 3C147 to get the J catalog names though.